src/index.d.ts

```diff
@@ -136,6 +136,7 @@ declare module 'minecraft-protocol' {
 		playerCount: number
 		maxPlayers: number
 		motd: string
```

Comment:
motd should be optional too

Reply:
The motd defaults to 'A Minecraft server' and is needed for legacy ping responses

Reply:
sounds good

Reply:
maybe note that in another pr